spring-kafka/src/test/java/org/springframework/kafka/listener/CommitOnAssignmentTests.java

```diff
@@ -135,13 +135,48 @@ void testLatestOnlyNoTx() throws InterruptedException {
 		KafkaTransactionManager tm = new KafkaTransactionManager<>(pf);
 		Producer producer = mock(Producer.class);
 		given(pf.createProducer(any())).willReturn(producer);
-		CountDownLatch latch = new CountDownLatch(1);
 		props.setTransactionManager(tm);
```

Comment:
Can we consider to not use deprecated API then?
Although it is not clear why deprecation of that property is the part of this change.
It is really hard to review mixed concerns.

Reply:
I'll create anthoer issue to this pr